lib/bolt_server/transport_app.rb

```diff
@@ -54,7 +54,13 @@ def initialize(config)
 
       body = JSON.parse(request.body.read)
       schema_error = JSON::Validator.fully_validate(@schemas["ssh-run_task"], body)
-      return [400, schema_error.join] if schema_error.any?
+      if schema_error.any?
+        msg = "There was an error validating the request body. Please see for expected schema"
```

Comment:
I don't think you need the `please see for expected schema'. See the details? 

Also in general would it be worth just having a `make_error(kind, msg, details)` method that returns error hash?

Reply:
Yeah, I'm not sure why I didn't use `Bolt::Error`....refactored!